### DISCUSSION TITLE
how to costamize

### DISCUSSION THREAD

**Discussion by SkveeRLS** [wg-easy/wg-easy#1341] [Q&A] [answered]
Hi, I would like to know how to costamize the web to add my functions - I can't find the files on the server.
thanks

**Comment by tetuaoro** [comment]
Is this related t*a* #1346 ?? Y*ao* can c*a*ntrib*o*te instead.

**Comment by SkveeRLS** [comment]
I mean redoing files and scripts and adding your own code.

**Reply by eandersons** [reply]
It is simple:

1. get project files either by:
   - forking the repository and then cloning it if you plan to create merge requests;
   - if you do not plan to create merge requests:
     - cloning the repository directly;
     - downloading ZIP archive;
2. make the desired changes;
3. build your custom Docker image;
4. run WireGuard Easy container with your custom image.

**Reply by SkveeRLS** [reply, marked as answer]
Thank you very much.
